cogs/ReactForRole.py

```diff
@@ -937,10 +937,11 @@ async def overwrite_channel_add_reaction_perms(self, guild: discord.Guild, chann
         """
         #  Get the @everyone role.
         role: discord.Role = discord.utils.get(guild.roles, id=guild.id)
-        overwrite: discord.PermissionOverwrite = discord.PermissionOverwrite()
-        overwrite.update(add_reactions=False)
-        await channel.set_permissions(role, overwrite=overwrite)
+        perms = channel.overwrites_for(role)
```

Comment:
All current roles should have the `add_reactions` perm set to false, not just everyone. (although this successfully doesn't screw with other perms, as mentioned in issue, gj!)